### PR TITLE
Update Post “2018-10-gnomad-v2-1”

### PR DESCRIPTION
**Pull request by ch-kr**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by ch-kr** [comment]
Updating v2.1 blog post to change "outbred population" to "non-bottlenecked population"

File: content/posts/2018-10-gnomad-v2-1.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ are 5 such arrays at the root of the row schema:
 - `freq`: Allele frequency information (AC, AN, AF, homozygote count)
 - `age_hist_het`: Histogram of heterozygous carriers
 - `age_hist_hom`: Histogram of homozygous carriers
-- `popmax`: Allele frequency information for the outbred population with the highest frequency.
+- `popmax`: Allele frequency information for the non-bottlenecked population with the highest frequency.
   This excludes Finns, Ashkenazi Jewish and “Other” populations.
```
Comment (jkgoodrich) [inline content/posts/2018-10-gnomad-v2-1.md:147]:
```suggestion
  This excludes Ashkenazi Jewish (asj), European Finnish (fin), and "Other" (oth) populations.
```

File: content/posts/2018-10-gnomad-v2-1.md [hunk 2]
```diff
@@ -533,3 +533,6 @@ George Grant, Sam Novod, Valentin Ruano-Rubio, and Bradley Taylor.
 
 And finally, special thanks to the Broad Institute Genomics Platform as a whole, who generated 90%
 of the data used in gnomAD, and also provided critical computing resources required for this work.
+
+
+Post has been edited on April 20, 2022 to rephrase "outbred population" as "non-bottlenecked population".
```
Comment (jkgoodrich) [inline content/posts/2018-10-gnomad-v2-1.md:538]:
```suggestion
*Post was updated on April 20, 2022 to rephrase "outbred population" as "non-bottlenecked population"*.
```

**Review by jkgoodrich — CHANGES_REQUESTED** [review]

**Review by jkgoodrich — APPROVED** [review]
LGTM, thank you for adding!

**Pull request merged**